src/Http/Headers/src/HeaderNames.cs

```diff
@@ -24,6 +24,7 @@ public static class HeaderNames
         public static readonly string AltSvc = "Alt-Svc";
         public static readonly string Authority = ":authority";
         public static readonly string Authorization = "Authorization";
+        public static readonly string Baggage = "baggage";
```

Comment:
All other headers are pascal case. This won't matter when getting the value from the headers collection - it is case-insensitive.
```suggestion
        public static readonly string Baggage = "Baggage";
```

Reply:
Yeah, I was curious about that. I kept it consistent with `traceparent` and `tracestate`, the other two headers related to this standard ([W3C Trace Context](https://www.w3.org/TR/trace-context/)) that are lowercase in this class.

Reply:
Ok. Perhaps better as is then.

Reply:
Since HTTP/2 all header names are officially lower case so it might make sense to start doing that with new entries. You're right though that the collections are all ignore-case so it shouldn't make a difference.